scripts/gen_screenshots.py

```
@@ -150,56 +149,65 @@ def make_room():
+    y += 46
+
+    # Peers card — linear list matching the actual PeerRow-based UI
+    peers = [
+        ("TR", "Taylor Rivera", PURPLE, True,  False),
+        ("MK", "Morgan Kim",    GREEN,  False, True),
+    ]
+    row_h = 120
+    card_total = len(peers) * row_h
+    rounded_rect(d, [PAD, y, W - PAD, y + card_total], r=16, fill=CARD_BG)
+
+    for i, (initials, name, col, talking, muted) in enumerate(peers):
```

Ruff (0.15.12)

[warning] Loop control variable `muted` not used within loop body

Rename unused `muted` to `_muted`

(B007)